### ISSUE TITLE
Switch: Minor aria-label updates

### ISSUE THREAD

**Issue by rebeccaalpert** [patternfly/patternfly-react#1602]
Per @jgiardino in https://github.com/patternfly/patternfly-react/pull/1591#issuecomment-474049414, the Switch component should have the following improvements:

> - For the examples that have visible text labels, the `aria-label` prop should be set to the same text string that displays for the `on` state. Specifically, the following examples should all have this prop defined with this value: 
>   `aria-label="Message when on"`:
>   - Simple Switch
>   - Disabled Switch, the first two `<Switch>` component instances
>   - Uncontrolled Switch, the first two `<Switch>` component instances
>   - Actually, looking at these examples again, can we just set `aria-label="Message when on"` for all of them?
> - Are you able to update the description for `aria-label` to include something like: 
>   > Adds accessible text to the Switch, and should describe the `isChecked="true"` state. When `label` is defined, `aria-label` should be set to the text string that is visible when `isChecked` is true.
